modules/simple_forms_api/app/controllers/simple_forms_api/v1/scanned_form_uploads_controller.rb

```diff
@@ -51,7 +56,43 @@ def validated_metadata
       end
 
       def upload_pdf(file_path, metadata)
-        SimpleFormsApi::PdfUploader.new(file_path, metadata, params[:form_number]).upload_to_benefits_intake(params)
+        location, uuid = prepare_for_upload
```

Comment:
Changes in this file should be extracted to the PdfUploader class but I want to do one thing at a time so it's a bit verbose for now.